### PR TITLE
Migrate Kubernetes To Cloud Config

### PR DESCRIPTION
**Pull request by commit-dkp**
Importing the instance_type, disk_type, and networks.

### CODE REVIEW

File: cloud-config/cloud-config-main.yml [hunk 1]
```diff
@@ -56,6 +56,21 @@ networks:
       subnet: (( grab terraform_outputs.services_subnet_az1 ))
       security_groups:
       - (( grab terraform_outputs.bosh_security_group ))
+- name: minion
```
Comment (jmcarp) [inline cloud-config/cloud-config-main.yml]: Why call this network `minion`? Unless this refactor is separating the deployment into multiple networks, all instance groups will live in the same network, which we'd probably call `kubernetes`.

File: cloud-config/cloud-config-main.yml [hunk 2]
```diff
@@ -148,6 +169,10 @@ disk_types:
   cloud_properties:
     type: gp2
     encrypted: true
+- name: kubernetes_storage_class
```
Comment (jmcarp) [inline cloud-config/cloud-config-main.yml]: See https://github.com/18F/cg-deploy-kubernetes/pull/71/files#r120035804. Kubernetes dynamic volumes are managed by kubernetes and not bosh, so we don't need this.

File: cloud-config/cloud-config-main.yml [hunk 3]
```diff
@@ -116,6 +131,12 @@ vm_types:
 - name: logsearch_errand
   cloud_properties:
     instance_type: t2.medium
+- name: minion
```
Comment (jmcarp) [inline cloud-config/cloud-config-main.yml]: Prefix with `kubernetes_` for clarity. Also, won't we also need `etcd` and `consul` types?

**Comment by wjwoodson** [comment]
Dependencies for https://github.com/18F/cg-deploy-kubernetes/pull/71, https://github.com/18F/kubernetes-release/pull/55

File: cloud-config/cloud-config-main.yml [hunk 4]
```diff
@@ -116,6 +116,21 @@ vm_types:
 - name: logsearch_errand
   cloud_properties:
     instance_type: t2.medium
+- name: kubernetes_default
```
Comment (jmcarp) [inline cloud-config/cloud-config-main.yml]: Could we use a more descriptive name?

File: cloud-config/cloud-config-main.yml [hunk 5]
```diff
@@ -167,6 +187,17 @@ vm_extensions:
 - name: logsearch-ingestor-profile
   cloud_properties:
     iam_instance_profile: (( grab terraform_outputs.logsearch_ingestor_profile ))
+- name: kubernetes-etcd-profile
+  cloud_properites:
+    iam_instance_profile: (( grab terraform_outputs.kubernetes_etcd_profile ))
```
Comment (jmcarp) [inline cloud-config/cloud-config-main.yml]: This doesn't seem to exist--looks like we want `etcd_backup_profile` instead.

**Pull request merged**